src/stravalib/client.py

```diff
@@ -656,11 +730,25 @@ def create_activity(
         ----------
         name : str
             The name of the activity.
-        activity_type : str
+        activity_type : str, default=None
```

Comment:
i set both of these "types" to optional. but we probably want to run a test on params that one or the other exists otherwise the user will get a bad call return

```python
stravalib.exc.Fault: 400 Client Error: Bad Request [Bad Request: [{'resource': 'Activity', 'field': 'type', 'code': 'missing_field'}]]
```
the above is useful but we could just run a check before the call is made and return an error then to be proactive.

Reply:
this has been addressed via a value error.